### PR TITLE
Correct doc sentence [ci skip]

### PR DESCRIPTION
**Pull request by mechanicles**

### CODE REVIEW

File: guides/source/active_record_querying.md [hunk 1]
```diff
@@ -1343,7 +1343,7 @@ Client.unscoped {
 Dynamic Finders
 ---------------
 
-For every field (also known as an attribute) you define in your table, Active Record provides a finder method. If you have a field called `first_name` on your `Client` model for example, you get `find_by_first_name` for free from Active Record. If you have a `locked` field on the `Client` model, you also get `find_by_locked` and methods.
+For every field (also known as an attribute) you define in your table, Active Record provides a finder method. If you have a field called `first_name` on your `Client` model for example, you get `find_by_first_name` for free from Active Record. If you have a `locked` field on the `Client` model, you also get `find_by_locked` method.
```
Comment (prathamesh-sonpatki) [inline guides/source/active_record_querying.md]: I think we can replace `and methods` with `and similar dynamic methods`. What do you think?

Comment (matthewd) [inline guides/source/active_record_querying.md]: I think this is fine; there are no other "similar" methods (are there?)

Comment (prathamesh-sonpatki) [inline guides/source/active_record_querying.md]: I mean to say `similar dynamic methods for all the other fields` :smiley:

Comment (mechanicles) [inline guides/source/active_record_querying.md]: @prathamesh-sonpatki Yup that would make more sense. I will add it later.

**Pull request merged**